circle/circle.go

```diff
@@ -20,4 +21,10 @@ func main() {
 	// ++ Your code here! ++
```

Comment:
You may remove `// ++ Your code here! ++` comments from the code. They are here just so you would know which part you need to modify.